### ISSUE TITLE
Does this extension also works with a local installation of influxdb?

### ISSUE THREAD

**Issue by chargerharry** [JoshNeedsAJob/InfluxCloudExtension#6]
Hello,
you write that the Influx cloud extension is intended to make executing SQL, InfluxQL and Flux queries against the cloud version of Influx easier.

I have a local installation of influx db.

Can the extension also be used in a local enviroment to write InfluxQL Queries?

Many thanks in advance for your answer.

Best regards

chargerharry

**Comment by JoshB-KC** [comment]
Yes.  This should work against local instances.  You will just need to put in the correct connection string.
